app/screens/home/home_screen.tsx

```diff
@@ -73,6 +83,22 @@ export default function HomeScreen({
     }
   }, [user.uid]);
 
+  useEffect(() => {
+    if (user.uid) {
```

Comment:
You should put if (user?.id) in case the user is not defined.

Reply:
Defensive programming is better in this case, and we don't fetch the groups if the user is not connected.